HousingManagementSystemApi/serverless.yml

```diff
@@ -0,0 +1,120 @@
+service: HousingManagementSystemApi
+provider:
+  name: aws
+  runtime: dotnet6
+  memorySize: 2048
+  tracing:
+    lambda: true
+    apiGateway: true
+  vpc: ${self:custom.vpc.${opt:stage}}
+  stage: ${opt:stage}
+  # TODO: Enable this line if DynamoDb is in use
+  #account: ${opt:account}
+  region: eu-west-2
+  apiKeys:
+    - secureAccess:
+        - api-key-${self:service}-${self:provider.stage}
+  usagePlan:
+    - secureAccess:
+        throttle:
+          burstLimit: 200
+          rateLimit: 100
+
+package:
+  artifact: ./bin/release/net6.0/HousingManagementSystemApi.zip
+
+functions:
+  HousingManagementSystemApi:
+    name: ${self:service}-${self:provider.stage}
+    handler: HousingManagementSystemApi::HousingManagementSystemApi.LambdaEntryPoint::FunctionHandlerAsync
+    role: lambdaExecutionRole
+    environment:
+      AUTHENTICATION_IDENTIFIER: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/authentication-identifier}
+      JWT_SECRET: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/jwt-secret}
+      UNIVERSAL_HOUSING_CONNECTION_STRING: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/universal-housing-connection-string}
```

Comment:
A `UNIVERSAL_HOUSING_CONNECTION_STRING` won't be required at Hackney and it would be best to not introduce one that could be forgotten to be removed.

The code consuming it (in [Startup.cs](https://github.com/LBHackney-IT/HousingManagementSystemApi/blob/development/HousingManagementSystemApi/Startup.cs#L37)) could be commented/removed to ensure an exception is not thrown.

Code changes will be required in this area when integrating with addresses and asset API, so the changes could be deferred until then. 

Will leave the choice to change now or at integration time with you.

Reply:
i have removed this now